File: backend/springnatour/src/main/java/com/unina/springnatour/dto/route/RouteTitleDto.java
package com.unina.springnatour.dto.route;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.io.Serializable;

@Data
public class RouteTitleDto implements Serializable {
    @JsonProperty("route_id")
    private Long routeId;

    @JsonProperty("route_title")
    private String routeTitle;
}

File: client/app/src/main/java/com/unina/natourkt/core/data/remote/dto/RouteTitleDto.kt
package com.unina.natourkt.core.data.remote.dto

import com.google.gson.annotations.SerializedName

data class RouteTitleDto(
    @SerializedName("route_id")
    val id: Long,

    @SerializedName("route_title")
    val title: String,
)

